pkg/flow/database.go

```diff
@@ -417,6 +440,16 @@ func (d *refData) rev() *ent.Revision {
 	return d.ref.Edges.Revision
 }
 
+func (d *refData) tags() map[string]string {
```

Comment:
Some structs contain other structs in a heirarchy. For example, if I'm remembering accurately: `instData` contains a `refData`, which contains a `wfData` which contains `nodeData`. 

You've created `tags()` functions for each of these objects that return only the tags relevant on that level. It might be a good idea for structs to first call the same method on the parent object and then append their own tags into the same map. What do you think?

Reply:
this issue should be resolved for now